ticket_booking_app/src/test/java/com/mordor/model/dto/validators/ReservationValidatorTest.java

```diff
@@ -0,0 +1,99 @@
+package com.mordor.model.dto.validators;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.validation.DataBinder;
+import org.springframework.validation.FieldError;
+import com.mordor.model.dto.ReservationDTO;
+import com.mordor.model.dto.TicketDTO;
+
+@SpringBootTest
+class ReservationValidatorTest {
+
+	private ReservationValidator validator = new ReservationValidator();
+	
+	private ReservationDTO reservationDTO = new ReservationDTO();
+	
+	private DataBinder binder;
+
+	@BeforeEach
+	public void setUp()  {
+		reservationDTO.setName("Michał");
+		reservationDTO.setSurname("Jasnowski");
+		
+		List<TicketDTO> tickets = new ArrayList<>();
+ 		TicketDTO ticket = new TicketDTO();
+ 		ticket.setSeatId((long) 1);
+ 		ticket.setScreeningId((long) 1);
+ 		ticket.setTicketTypeId((long) 1);
+		tickets.add(ticket);
+		reservationDTO.setTickets(tickets);
+		
+		binder = new DataBinder(reservationDTO);
+		binder.setValidator(validator);
+	}
+	
+	@Test
+	void testNameValidation() {
+		FieldError fe;
+		binder.validate();
+		fe = binder.getBindingResult().getFieldError();
+		assertNull(fe);
+		
+		setUp();

```

Comment:
Jeśli musisz wołać tą metodę, to może lepiej rozbić ten test na kilka mniejszych, ew. użyć parametrów?